path_config.go

```diff
@@ -11,44 +11,52 @@ func pathConfig(b *azureAuthBackend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config",
 		Fields: map[string]*framework.FieldSchema{
-			"tenant_id": &framework.FieldSchema{
+			"tenant_id": {
 				Type: framework.TypeString,
 				Description: `The tenant id for the Azure Active Directory.  This is sometimes
-				referred to as Directory ID in AD.  This value can also be provided with the 
-				AZURE_TENANT_ID environment variable.`,
+							referred to as Directory ID in AD.  This value can also be provided with the 
```

Comment:
Can you remove the extra indents in here (same for all the param descriptions below)? This is rendered as well when doing `vault path-help`:
```
## PARAMETERS

    client_id (string)

        The OAuth2 client id to connection to Azure.
        							This value can also be provided with the AZURE_CLIENT_ID environment variable.

    client_secret (string)

        The OAuth2 client secret to connection to Azure.
        							This value can also be provided with the AZURE_CLIENT_SECRET environment variable.

    environment (string)

        The Azure environment name. If not provided, AzurePublicCloud is used.
        							This value can also be provided with the AZURE_ENVIRONMENT environment variable.

    resource (string)

        The resource URL for the vault application in Azure Active Directory.
        							This value can also be provided with the AZURE_AD_RESOURCE environment variable.

    tenant_id (string)

        The tenant id for the Azure Active Directory.  This is sometimes
        							referred to as Directory ID in AD.  This value can also be provided with the
        							AZURE_TENANT_ID environment variable.
```

```suggestion
referred to as Directory ID in AD.  This value can also be provided with the 
```